x-pack/plugins/security_solution/public/common/components/endpoint/formatted_date_time.tsx

```diff
@@ -8,10 +8,14 @@
 import React from 'react';
 import { FormattedDate, FormattedTime, FormattedRelative } from '@kbn/i18n/react';
 
-export const FormattedDateAndTime: React.FC<{ date: Date }> = ({ date }) => {
+export const FormattedDateAndTime: React.FC<{ date: Date; showRelativeTime?: boolean }> = ({
+  date,
+  showRelativeTime = false,
```

Comment:
Why add the new prop?  do you want relative time to always be shown? if so, you might just want to use `<FormattedRelativeTime>` directly in your component. But if that is the case, I would also talk to Bonnie about that - always showing relative time (IMO) is not a good after the first 1h.

Reply:
Ah, I see what you mean. I guess I misunderstood you earlier. I assumed that we always want to show relative time. At least that's what it looked like in the mocks. This UX is still going to be worked on more in subsequent PRs. So we've time to make this slicker. 🙏